express-hello-world/server.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+const app = express();
```

Comment:
Put a blank line after your `import` statements, and another after creating `app`. This will make the code easier to follow -> all the imports grouped together, all your configuration (we will be adding more) grouped together, all your endpoints grouped together, and then finally `listen`.